tests/tensor_search/integ_tests/test_search_structured.py

```diff
@@ -1332,3 +1334,15 @@ def test_filter_on_large_integer_and_float(self):
                     self.assertEqual(1, len(res["hits"]))
                     self.assertEqual(expected_document_ids, res["hits"][0]["_id"])
 
+    def test_tensor_search_query_can_be_none(self):
+        res = tensor_search.search(text=None, config=self.config, index_name=self.default_text_index,
+                                   context=SearchContext(
+                                       **{"tensor": [{"vector": [1, ] * 384, "weight": 1},
+                                                     {"vector": [2, ] * 384, "weight": 2}]}))
+
+        self.assertIn("hits", res)
+
+    def test_lexical_query_can_not_be_none(self):
+        with self.assertRaises(InvalidArgError):
+            res = tensor_search.search(text=None, config=self.config, index_name=self.default_text_index,
```

Comment:
Try adding `context` here to prove it still errors out.

Reply:
Same comment with the unstructured test.

Reply:
done